crates/nu-utils/src/sample_config/default_env.nu

```diff
@@ -44,13 +44,13 @@ def create_right_prompt [] {
 
 # Use nushell functions to define your right and left prompt
 $env.PROMPT_COMMAND = {|| create_left_prompt }
-$env.PROMPT_COMMAND_RIGHT = {|| create_right_prompt }
+# $env.PROMPT_COMMAND_RIGHT = {|| create_right_prompt }
```

Comment:
why is the right prompt commented out?

Reply:
the two prompts are not the same :thinking: 

not setting the `PROMPT_COMMAND_RIGHT` was the easiest because it gives the same prompt as in `nu -n` :ok_hand: 

one other solution would be to change the `create_right_prompt` command or change the Rust implementation :thinking: 
i went with commenting out because it does the job and is much simpler :innocent:

Reply:
ya, @nushell/core-team needs to decide whether to change the reedline rust code to match the new prompt, or change the default_env.nu to match the latest config changes. I'm not sure. I think @jntrnr made changes recently because MM/DD/YYYY may be confusing to people so jt change it to YYYY-MM-DD iirc.

Reply:
i'm fine either way :+1: 

this change makes both the prompts the same at least.